share/man/man9/intro.9

```diff
@@ -133,8 +133,9 @@ There is also
 .Ss Memory Management
 Dynamic memory allocations inside the kernel are generally done using
 .Xr malloc 9 .
-Frequently allocated objects may prefer to use
-.Xr uma 9 .
+For frequently allocated objects,
+.Xr uma 9
+may be preferable.
```

Comment:
I don't think this sentence needs to change, unless someone is going to add more detail.